src/textinput/example/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Text-based Input Example</title>
+</head>
+<body>
+	<script src="../../../../node_modules/@dojo/loader/loader.min.js"></script>
+	<link rel="stylesheet" href="../../common/styles/widgets.css">
```

Comment:
The other widgets have been using `widgets.css` in their example files. If we change it, we should probably change all of them